### PR TITLE
#495: Alter history only when configured to do so

### PR DESCRIPTION
**Pull request by scorphus**

### CODE REVIEW

File: thefuck/shells/fish.py [hunk 1]
```diff
@@ -18,17 +19,20 @@ def _get_overridden_aliases(self):
...
         # It is VERY important to have the variables declared WITHIN the alias
         return ('function {0} -d "Correct your previous console command"\n'
                 '  set -l fucked_up_command $history[1]\n'
                 '  env TF_ALIAS={0} PYTHONIOENCODING=utf-8'
                 ' thefuck $fucked_up_command | read -l unfucked_command\n'
                 '  if [ "$unfucked_command" != "" ]\n'
                 '    eval $unfucked_command\n'
-                '    history --delete $fucked_up_command\n'
-                '    history --merge ^ /dev/null\n'
+                '    history --delete $fucked_up_command\n{1}'
```
Comment (nvbn) [inline thefuck/shells/fish.py]: Maybe we shouldn't delete entry from history without `alter_history`?

Comment (scorphus) [inline thefuck/shells/fish.py]: Oops! Absolutely!

Comment (nvbn) [inline thefuck/shells/fish.py]: Thanks =)

**Comment by scorphus** [comment]
Thanks for pointing that out, @nvbn :+1:

**Comment by nvbn** [comment]
Thanks!

**Pull request merged**